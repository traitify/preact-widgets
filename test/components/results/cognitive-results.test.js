import Component from "components/results/cognitive-results";
import ComponentHandler from "support/component-handler";

describe("CognitiveResults", () => {
  it("renders components", () => {
    const component = new ComponentHandler(<Component />);

    expect(component.tree).toMatchSnapshot();
  });
});
