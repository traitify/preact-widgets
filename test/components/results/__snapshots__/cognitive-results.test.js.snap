// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CognitiveResults renders components 1`] = `
<section>
  <img
    alt="Brain"
    src="https://cdn.traitify.com/images/cognitive/brain.png"
  />
  <h1>
    Thank you for taking the assessment.
  </h1>
  <p>
    To learn more about the assessment you just took, please visit 
    <a
      href="https://www.traitify.com"
    >
      Traitify.com
    </a>
    .
  </p>
</section>
`;
